ext/drx/drx.c

```diff
@@ -2297,6 +2304,12 @@ drx_expand_scatter_gather(void *drcontext, instrlist_t *bb, OUT bool *expanded)
         return true;
     if (!instr_is_gather(first_app) && !instr_is_scatter(first_app))
         return true;
+
+    /* We want to avoid spill slot conflicts with later instrumentation passes. */
```

Comment:
See earlier comment: also need to adjust total spill slots requested.  We want to avoid moving into the expensive DR non-direct slots.

Reply:
`drx_expand_scatter_gather` already does that.